test/functional/VM_Test/src/j9vm/test/arraycopy/MultiThreadedBackwardArrayCopyTest.java

```diff
@@ -28,84 +28,84 @@
+		createValidValue(27),
+		createValidValue(28),
+		createValidValue(29),
+		createValidValue(30),
+		createValidValue(31),
+		createValidValue(32),
+		createValidValue(33),
+		createValidValue(34),
+	};
 
-   static HashSet<Object> validHashSet = new HashSet<Object>(Arrays.asList(source));
-   String name;
+	static HashSet<Object> validHashSet = new HashSet<Object>(Arrays.asList(source));
+	String name;
 	static int count = 1000000;
-	MultiThreadedBackwardArrayCopyTest(String name){
-      this.name = name;
-   }
+	MultiThreadedBackwardArrayCopyTest(String name) {
+		this.name = name;
+	}
 
-   public static void main(String[] args) throws InterruptedException {
-      new Thread(new MultiThreadedBackwardArrayCopyTest("Thread-1")).start();
-      new Thread(new MultiThreadedBackwardArrayCopyTest("Thread-2")).start();
-      new Thread(new MultiThreadedBackwardArrayCopyTest("Thread-3")).start();
-      new Thread(new MultiThreadedBackwardArrayCopyTest("Thread-4")).start();
-   }
+	public static void main(String[] args) throws InterruptedException {
+		new Thread(new MultiThreadedBackwardArrayCopyTest("Thread-1")).start();
+		new Thread(new MultiThreadedBackwardArrayCopyTest("Thread-2")).start();
+		new Thread(new MultiThreadedBackwardArrayCopyTest("Thread-3")).start();
+		new Thread(new MultiThreadedBackwardArrayCopyTest("Thread-4")).start();
+	}
 
-   public void run(){
-      Object[] dest = new Object[source.length];
-      for (int i=0; i < count; i++){
-         rotateSource();
-         System.arraycopy(source, 0, dest, 0, dest.length);
-         for (int j=0; j<dest.length; j++){
-            if (!validHashSet.contains(dest[j])){
-               System.out.println("Error: Found unexpected object in array at "+j+" in "+name);
-            }
-         }
-      }
-   }
+	public void run() {
+		Object[] dest = new Object[source.length];
+		for (int i=0; i < count; i++) {
+			rotateSource();
+			System.arraycopy(source, 0, dest, 0, dest.length);
+			for (int j=0; j<dest.length; j++){
+				if (!validHashSet.contains(dest[j])){
+					System.out.println("Error: Found unexpected object in array at "+j+" in "+name);
+				}
+			}
+		}
+	}
 
 
 	// Create a unique String object
-   private static Object createValidValue(int i){
+	private static Object createValidValue(int i) {
 		// Create array of bytes to spreadout the objects
-      byte [] unused = new byte[17832];
-      return Integer.toString(i);
-   }
+		byte [] unused = new byte[17832];
+		return Integer.toString(i);
+	}
 
 	// Rotate source right forcing it to do array copy in backward direction
-   public static void rotateSource(){
-      Object temp = source[source.length-1]; // Getting last element in temp
-      System.arraycopy(source, 0, source, 1, source.length-1);
-      source[0] = temp;
-   }
-
+	public static void rotateSource() {
+		Object temp = source[source.length-1]; // Getting last element in temp
+		System.arraycopy(source, 1, source, 2, source.length-2);
```

Comment:
Spaces needed around arithmetic operators here.

Reply:
Changed in https://github.com/eclipse/openj9/pull/3623/commits/254a0edac69d3b5254e783c2997c388a1a5055e6